### DISCUSSION TITLE
Stack Synchronisation Between Multiple Tabs

### DISCUSSION THREAD

**Discussion by moldywart** [nroduit/Weasis#566] [Q&A]
Hi, thank you for creating such a helpful DICOM viewer. 

I have a few different studies of the same subject that I would like to stack sync while I scroll. 

Is it possible to synchronise two studies (same subject) done at different times? When I select stack sync, only the series done in the same study can sync, ie I cannot sync the study done in January and the one done in April. It’s the same issue whether I open the studies in the same tab, or in different tabs. 

Is it possible to sync studies of different times, as the comparison between two different studies of the same subject would be infinitely helpful.

**Comment by nroduit** [comment]
Series synchronization has nothing to do with open tabs or series dates.

By default, [Synchronize](https://weasis.org/en/tutorials/dicom-2d-viewer/index.html#toolbars) is set to "Default Stack", and a shared Frame of Reference UID (0020,0052) is used to determine a common 3D coordinate system. Otherwise, the relationship between coordinate systems is uncertain and would have to be calibrated manually, which Weasis does not allow.
